<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_add_assignment">
       <property name="text">
        <string>Add Assingment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete_assignment">
       <property name="text">
        <string>Delete Assignment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_selected_assignment">
       <property name="text">
        <string>Selected Assignment</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Roboto Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Assignments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_assignments"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Roboto Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Done</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_done"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
